luzer/CMakeLists.txt

```diff
@@ -1,7 +1,19 @@
 SetHwArchString(ARCH)
-set(LIBFUZZER_NO_MAIN_LIB_NAME "libclang_rt.fuzzer_no_main-${ARCH}.a")
+if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
+  set(LIBFUZZER_NO_MAIN_LIB_NAME "libclang_rt.fuzzer_no_main-${ARCH}.a")
+elseif (CMAKE_SYSTEM_NAME STREQUAL "Darwin")
+  set(LIBFUZZER_NO_MAIN_LIB_NAME "libclang_rt.fuzzer_no_main_osx.a")
+else()
+  message(FATAL_ERROR "Unsupported system: ${CMAKE_SYSTEM_NAME}")
+endif()
+
+SetClangLibPath("libclang_rt.asan-${ARCH}.a" LIBCLANG_ASAN_LIB)
+SetClangLibPath("libclang_rt.ubsan_standalone-${ARCH}.a" LIBCLANG_UBSAN_LIB)
+SetClangLibPath("libclang_rt.ubsan_standalone_cxx-${ARCH}.a" LIBCLANG_UBSAN_CXX_LIB)
 SetClangLibPath(${LIBFUZZER_NO_MAIN_LIB_NAME} FUZZER_NO_MAIN_LIBRARY)
 
+include(BuildLibFuzzer)
```

Comment:
Nit: I would rather name it `BuildSanitizers` or whatsoever (since it is more about sanitizers configuration).